js/src/frameworks/langchain.ts

```diff
@@ -37,21 +33,25 @@ export class LangchainToolSet extends BaseComposioToolSet {
       runtime: config?.runtime || LangchainToolSet.FRAMEWORK_NAME,
       entityId: config.entityId || LangchainToolSet.DEFAULT_ENTITY_ID,
     });
+    this.connectedAccountIds = config.connectedAccountIds || {};
```

Comment:
Move this baseToolset. connectedAccountsIds, shouldn't record. We should create a record and throw the error in case an invalid id is passed.

Reply:
1. Need to move connectedAccountIds to base.toolset class
2. Inside base.toolset if connectedAccountIds is passed then need to check if each connectedAccount if valid or not?